contracts/LifCrowdsale.sol

```diff
@@ -126,10 +137,17 @@ contract LifCrowdsale is Ownable, Pausable {
     TokenPurchase(msg.sender, beneficiary, weiAmount, tokens);
   }
 
-  function addPresaleTokens(address beneficiary, uint256 tokens) onlyOwner {
+  function addPrivatePresaleTokens(address beneficiary, uint256 totalWei) onlyOwner {
     require(block.number < startBlock);
     require(beneficiary != address(0));
-    require(tokens > 0);
+    require(totalWei > 0);
```

Comment:
I think we should change the naming here. `totalWei` in this function is the amount of wei that corresponds to this presale payment. But `maxPresaleWei` is the sum of the wei sent in all the presale payments. The naming doesn't help in understanding the relation between those two values. I think we should rename `totalWei` to `weiSent` and `totalPresaleWei` to `totalPresaleWeiReceived`

What do you think @AugustoL?